libcrux_crypto/Cargo.toml

```diff
@@ -10,9 +10,18 @@ repository = "https://github.com/openmls/openmls/tree/main/openmls_libcrux_crypt
 readme = "../README.md"
 
 [dependencies]
-getrandom = "0.2.12"
-libcrux = { version = "=0.0.2-alpha.3", features = ["rand"] }
+libcrux-chacha20poly1305 = { version = "0.0.2" }
+libcrux-ed25519 = { version = "0.0.2", features=["rand"] }
+libcrux-hkdf = { version = "0.0.2" }
+libcrux-sha2 = { version = "0.0.2" }
 openmls_traits = { version = "0.3.0", path = "../traits" }
 openmls_memory_storage = { version = "0.3.0", path = "../memory_storage" }
-rand = "0.8.5"
+rand = "0.9.0"
 tls_codec.workspace = true
+rand_chacha = "0.9.0"
+
+hpke_rs = { git = "https://github.com/cryspen/hpke-rs", branch = "main", package = "hpke-rs" , features=["hazmat", "serialization"] }
```

Comment:
Let's make sure all of these dependencies have a version as well so that we can release this later.